dockerfiles/ubuntu18/build_custom/README.md

```diff
@@ -0,0 +1,71 @@
+# Build custom Intel® Distribution of OpenVINO™ toolkit Docker image
+This repository folder contains Dockerfiles to build an docker image with the Intel® Distribution of OpenVINO™ toolkit.  
```

Comment:
`This repository folder contains Dockerfile to build a docker image with the Intel® Distribution of OpenVINO™ toolkit.`